sql/core/src/test/resources/sql-tests/results/change-column.sql.out

```diff
@@ -27,11 +27,11 @@ struct<>
 -- !query output
 org.apache.spark.sql.catalyst.parser.ParseException
 
-Operation not allowed: ALTER TABLE table CHANGE COLUMN requires a TYPE or a COMMENT or a FIRST/AFTER(line 1, pos 0)
```

Comment:
The previous error message is better. Maybe we should make `alterColumnAction` optional, and throw error with nice  message if it's not specified.

Reply:
OK. I will update it.

FYI, in Postgres:
```SQL
postgres=# alter table t alter column i;
2020-02-05 14:26:22.056 EST [86874] ERROR:  syntax error at or near ";" at character 29
2020-02-05 14:26:22.056 EST [86874] STATEMENT:  alter table t alter column i;
ERROR:  syntax error at or near ";"
LINE 1: alter table t alter column i;
```